.travis.yml

```diff
@@ -172,8 +172,8 @@ script:
       else
         USE_DIRS="mne/";
       fi;
-    - echo py.test -m "\"${CONDITION}\"" ${OPTION} ${USE_DIRS};
-    - py.test -m "\"${CONDITION}\"" ${OPTION} ${USE_DIRS};
+    - echo py.test -m "\"${CONDITION}\"" ${OPTION} ${USE_DIRS} --cov=mne;
+    - py.test -m "\"${CONDITION}\"" ${OPTION} ${USE_DIRS} --cov=mne;
```

Comment:
It took me a while to figure out why just running a single test would go quickly but then hang for about 5 seconds before returning control to the terminal. Always using `--cov` is bad for local code testing, so let's only run it on the CIs.